src/public/js/_.js

```diff
@@ -1018,6 +1019,8 @@ module.exports = {
 				return 'G-Core';
 			case 'QT':
 				return 'Quantil';
+			case 'BY':
```

Comment:
Import `src/public/json/net-providers.json` where you need this and don't duplicate it anywhere else.

Reply:
See e.g. `src/views/components/network-provider-stats.html:96`

Reply:
Yes, you're right, i will review my code in order to use net-providers.json, once i am done with the Map element

Reply:
Done